ext/ffi/callback.rs

```diff
@@ -555,13 +543,31 @@ where
   let v8_value = cb.v8_value;
   let cb = v8::Local::<v8::Function>::try_from(v8_value)?;
 
-  let isolate: *mut v8::Isolate = &mut *scope as &mut v8::Isolate;
-  LOCAL_ISOLATE_POINTER.with(|s| {
-    if s.borrow().is_null() {
-      s.replace(isolate);
+  let isolate_id_result: Result<u32, u32> = LOCAL_ISOLATE_ID.with(|s| {
+    let value = *s.borrow();
+    if value == 0 {
+      let res = ISOLATE_ID_COUNTER.fetch_update(
```

Comment:
Is there any reason this can't be `fetch_add`?

Reply:
I tried using it but was getting 0 for some reason. But, I'll try again as it was probably just me failing to do it properly.

Reply:
Changed to use `fetch_add`. I was just failing earlier somehow.